### DISCUSSION TITLE
Image could not load

### DISCUSSION THREAD

**Discussion by enri90** [vikejs/vike#819] [Discussion] [answered]
do you need to do any special configuration for the images?

import NotFoundImage from '@/src/images/404-illustration.svg'

[vite:asset] Could not load /src/src/images/404-illustration.svg (imported by pages/_error/+Page.tsx): ENOENT: no such file or directory, open '/src/src/images/404-illustration.svg'
error during build:
Error: Could not load /src/src/images/404-illustration.svg (imported by pages/_error/+Page.tsx): ENOENT: no such file or directory, open '/src/src/images/404-illustration.svg'

```typescript
 resolve: {
    alias: {
      '@': path.resolve(__dirname, '/src')
    }
  },
``` 

```typescript
 "compilerOptions": {
    "strict": true,
    "module": "ES2020",
    "target": "ES2020",
    "moduleResolution": "Node",
    "lib": ["DOM", "DOM.Iterable", "ESNext"],
    "types": ["vite/client"],
    "jsx": "react-jsx",
    "skipLibCheck": true,
    "esModuleInterop": true,
    "baseUrl": ".",
    "paths": {
      "@/*": ["src/*"]
    }
  },
```

**Comment by brillout** [comment]
I believe it's more of a Vite question than a vite-plugin-ssr one. But let me know if you believe there is something specific about VPS.

**Comment by enri90** [comment, marked as answer]
false alarm

I got the path wrong, I shouldn't have added src

thx

**Reply by brillout** [reply]
Make sure to double check before asking. Thanks!

Thanks for the V1 design bug reports, keep them coming!
